openapi-generator/templates/java/libraries/okhttp-gson/build.gradle.mustache

```diff
@@ -142,6 +142,7 @@ dependencies {
     implementation 'com.squareup.okhttp3:logging-interceptor:3.14.7'
     implementation 'com.google.code.gson:gson:2.8.6'
     implementation 'io.gsonfire:gson-fire:1.8.4'
+    implementation 'org.openapitools:jackson-databind-nullable:0.2.1'
```

Comment:
The reason for adding this line was failure encountered in [Java tests](https://github.com/phrase/openapi/actions/runs/7755448581/job/21150828820) regarding package resolution.